flink-next-gen-connector-influxdb/src/test/java/org/apache/flink/streaming/connectors/InfluxDBSourceITCase.java

```diff
@@ -66,9 +69,14 @@ void testIncrementPipeline() throws Exception {
 
         CollectSink.VALUES.clear();
 
+        final Set<String> whiteList = new HashSet();
+        whiteList.add("test");
```

Comment:
The whitelist will be the main point we need to focus on next (in my opinion). Unfortunately I don't think the base implementation from druid will work well for us:

* It is sensible that the pipeline developer assigns a whitelist of fields they expect.
* However, this needs to be handled gracefully by the InfluxDB source. Two options:
  * Discard any measurements with non-whitelisted fields. Catch ParseException "Metric not whitelisted", return 204, done. Problem: the schema is not controllable from the pipeline point of view. If it changes without notice, the whole InfluxDB source breaks since no more data points are ingested.
  * Remove non-whitelisted fields from the measurement, send 204 → sensible, because it limits resource usage and still allows known fields to be passed along

I strongly prefer the second option. I would like to implement that without adding the whole parser lib, maybe we can just reuse the generated classes from druid if package visibility allows…